iot-hub/Samples/service/PnpServiceSamples/TemperatureController/TemperatureControllerSample.cs

```diff
@@ -114,20 +115,16 @@ private async Task InvokeRebootCommandAsync()
 
         private async Task UpdateDigitalTwinComponentPropertyAsync()
         {
+            
+            const string targetTemperaturePropertyName = "targetTemperature";
+
             // Choose a random value to assign to the targetTemperature property in thermostat1 component
             int desiredTargetTemperature = Random.Next(0, 100);
 
-            const string targetTemperaturePropertyName = "targetTemperature";
-            string propertyUpdate = CreatePropertyPatch(targetTemperaturePropertyName,
-                JsonConvert.SerializeObject(desiredTargetTemperature),
-                Thermostat1Component);
-            var twinPatch = new Twin();
-            twinPatch.Properties.Desired[targetTemperaturePropertyName] = desiredTargetTemperature;
+            var twinPatch = CreatePropertyPatch(targetTemperaturePropertyName, desiredTargetTemperature, Thermostat1Component);
+            _logger.LogDebug($"Update the {targetTemperaturePropertyName} property under component {Thermostat1Component} on the {_digitalTwinId} digital twin to { desiredTargetTemperature}.");
```

Comment:
Verb tense should be "Updating" or "Updated" depending on what tense we're following in the sample.